ui/tests/integration/components/pki/keys/form-test.js

```diff
@@ -46,7 +46,7 @@ module('Integration | Component | pki key form', function (hooks) {
     );
     assert.strictEqual(
       formErrorCount.innerText,
-      'There are 2 errors with this form.',
+      'There are 2 errors with this form. ',
```

Comment:
It looks like the test is failing now, so maybe this splace should be removed?
<img width="781" alt="Screen Shot 2022-12-14 at 6 36 05 PM" src="https://user-images.githubusercontent.com/68122737/207759116-bfaf9fb3-0c33-4a0f-834f-fae8829b917a.png">

Reply:
UGH this test! I keep adding and removing the space at the end and it keeps failing the other way 😒

Reply:
🤯 - maybe we could add `trimI()` in the test? 

```js
  assert.strictEqual(
      formErrorCount.innerText.trim(),
      'There are 2 errors with this form.',
      'renders correct form error count'
    );

Reply:
Oh never mind! I see how you refactored 😄